rb_ws/src/buggy/scripts/2d_sim/engine.py

```diff
@@ -227,8 +234,8 @@ def publish(self):
         heading_noisy = heading
 
         if (Simulator.NOISE):
-            lat_noisy = lat + np.random.normal(0, 1e-8)  # ~.1cm error
-            long_noisy = long + np.random.normal(0, 1e-8)  # ~.1cm error
+            lat_noisy = lat + np.random.normal(0, 1e-6)  # ~10cm error
```

Comment:
If our current error on the actual system is 0.5m, shouldn't the noise be more reflective of that?

Reply:
this is Gaussian noise, are you referring to the controller error? if so that is unrelated.